themes/addons/templates/Layout/Addons.ss

```diff
@@ -1,5 +1,5 @@
 <div class="page-header">
-	<h1>Modules and Themes <a class="pull-right" href="rss"><img src="themes/addons/images/feed-icon-28x28.png" width="20" alt="RSS Feed" /></a></h1>
+	<h1>Modules and Themes <a class="pull-right" href="/add-ons/rss"><img src="themes/addons/images/feed-icon-28x28.png" width="20" alt="RSS Feed" /></a></h1>

```

Comment:
Yes, or update the link method on the controller to take an action. Either will work